components/content/Callout.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="bg-surface-secondary rounded-lg">
+    <div class="flex p-5 space-x-5">
```

Comment:
Assuming you stole this from Tailwind but could we combine these two top-level divs to reduce nesting?